quaint/Cargo.toml

```diff
@@ -76,6 +76,7 @@ metrics = "0.18"
 futures = "0.3"
 url = "2.1"
 hex = "0.4"
+itertools = "0.10"
```

Comment:
Should we maybe move to workspace dependencies?

Also, I used the same version as other crates do, but we could update it to `0.12` FWIW.

Reply:
Good point but let's not worry about that in this PR - Could you maybe create a tech debt issue?

Reply:
https://github.com/prisma/team-orm/issues/750